i18n.module

```diff
@@ -260,7 +260,7 @@ function i18n_langcode($langcode = NULL) {
 /**
  * Implements hook_help().
```

Comment:
This needs evaluation. hook_help exists in Drupal 7, but got dropped in Backdrop. So currently this code isn't run at all.

Reply:
yea; I think we should remove this `hook_help`

Reply:
What we actually should do is to move the help text to a proper place (form). And we should do that also for all the submodules that currently still have an implementation of this hook. I'd prefer to leave this untouched for the moment.

Reply:
got it.

Reply:
I put it back.